### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by scottcreynolds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,147 @@
...
+To start, go to https://github.com/settings/tokens and click "Generate
+new token." Name it something like "Learn.co", and check `repo` scope. Once you generate
+the token, make sure to copy and paste it somewhere, because once you
+leave that page, you won't be able to see it again.
+
+Using the token to [access the API](https://developer.github.com/v3/oauth/#3-use-the-access-token-to-access-the-api) is a simple matter of creating an `Authorization` header with our request.
+
+We need to provide our authorization token in order to list our own
+repositories with this API, so let's add our `Authorization` header
+(don't forget to assign your token to `const token`).
```
Comment (pletcher) [inline README.md:59]: Would it be okay to use [`dotenv()`](https://www.npmjs.com/package/dotenv) for this?

We can set up the boilerplate for the students and just walk them through it quickly.

**Comment by pletcher** [comment]
I really like this lesson. The one thing is finding a way to exclude students' tokens from their committed results -- I think we can `dotenv()`-ify it. Would that work?

**Comment by scottcreynolds** [comment]
@pletcher would an environment variable work for client side js running in the browser when they try the code out? i didn't want them to have to jump through any hoops to see it in action. my gut says it muddies the message to never commit this stuff but willing to add it if you feel strongly?

**Comment by pletcher** [comment]
Ah shoot, you're right. (I'm way too used to having a build step.) I think you're right here -- let's keep it how you've written it :)

**Pull request merged**